nodejs/week1/week1homework/src/bakckend/routes/meals.js

```diff
@@ -0,0 +1,14 @@
+
+function reviewMeals() {
+    const  mealArr = require("../data/meals.json");
+    const reviewArr = require("../data/reviews.json");
+
+    return mealArr.map((meal)=>{
+        meal.reviews = reviewArr.filter((review) => review.mealId === meal.id);
+       return meal;
+    })
+
+}
```

Comment:
Fix formatting in this file.
You also have the opportunity to reuse getMealsWithReviews.